rosalind_ini6.py

```diff
@@ -1,11 +1,10 @@
 def main(input):
-    words = input.split()
     results ={}
```

Comment:
Eh, you shouldn't do it all in one line like that, even though it may be possible.

You have a couple options.

1. Use a vanilla dictionary, and solve it the old fashioned way like you're doing here.
2. https://docs.python.org/2/library/collections.html#defaultdict-examples
3. Why do number two when there's https://docs.python.org/2/library/collections.html#collections.Counter

Number one is the "right answer" (you solve it yourself using just the basics), number two is kinda meh. Number three is the _actual_ correct answer, but `Counter` is so powerful that this whole problem becomes a couple lines of code. There is no such things as "cheating", per say, but this gets pretty close. You do none of the work yourself, except for the formatting of the output and parsing of the input.

```py
def main(input):
    words = Counter(input.split())
    return '\n'.join([' '.join([k, str(v)]) for k, v in words.iteritems()])
```